storage/tail/28-tail.js

```diff
@@ -46,7 +47,29 @@ module.exports = function(RED) {
             }
         }
 
-        fileTail();
+        if (node.filename !== "") {
+            node.status({});
+            fileTail();
+        } else {
+            node.status({ fill: "grey", text: "paused" });
+            node.on('input', function (msg) {
+                if (!msg.hasOwnProperty("filename")) {
+                    node.error("Missing filename on input");
+                }else if (msg.filename === node.filename) {
+                    node.warn("No change of filename input");
```

Comment:
Not sure this needs to be a warm. If you have paused it for some reason you would want this to restart

Reply:
I'made it as a warning because the user should not think of it as needing an injection to do the tailing.
It works so feeding the node with an empty string will stop the tailing, and feeding it again with at new value/file would start tailing the new file. Feeding the node with the same value twice serves therefor no purpose, since the tail is already active.